New version of FastAPI breaks Titiler

<!-- Please search existing issues to avoid creating duplicates. -->


#### Problem description

A new version of FastAPI (v0.95.0) came out yesterday, and I've found that it breaks Titiler and stops it from running. 
When I run `uvicorn titiler.application.main:app` I get:

`AssertionError: Cannot use `Query` for path param 'TileMatrixSetId'`.

Installing FastAPI v0.94.1 fixes the issue

#### Expected Output

A running titiler server


#### Environment Information

MacOS Ventura, Conda/Mamba, python 3.11

Steps to recreate error:
```
mamba create -n titilererror python=3.11 -y
mamba activate titilererrror
pip install titiler.application fastapi==0.95.0
uvicorn titiler.application.main:app
```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 0.11.4 (2023-03-20)
+
+* set FastAPI version lower than 0.95 (https://github.com/tiangolo/fastapi/discussions/9278)
+
 ## 0.11.3 (2023-03-14)
 
 * handle dateline crossing dataset in COG/STAC Viewer

--- a/src/titiler/core/pyproject.toml
+++ b/src/titiler/core/pyproject.toml
@@ -29,7 +29,7 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-    "fastapi>=0.87.0",
+    "fastapi>=0.87.0,<0.95",
     "geojson-pydantic",
     "jinja2>=2.11.2,<4.0.0",
     "numpy",

